hyperspy/model.py

```diff
@@ -1251,7 +1251,7 @@ def fit(self, fitter="leastsq", method='ls', grad=False,
 
     def multifit(self, mask=None, fetch_only_fixed=False,
                  autosave=False, autosave_every=10, show_progressbar=None,
-                 interactive_plot=False, **kwargs):
+                 interactive_plot=False, iterpath='serpentine', **kwargs):
```

Comment:
We can only make ``serpentine`` the default ``iterpath`` method in 2.0. Otherwise, the result of multifitting would be unexpectedly different when using the next point release of HyperSpy.

Reply:
Now that you are at this, it would be cool if ``iterpath`` could take an iterator in addition to ``serpentine`` and ``flyback``.